src/components/Articles.jsx

```diff
@@ -1,13 +1,28 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+
 import ArticleList from './Lists/ArticleList';
```

Comment:
In this file and another one, I had to fix this import as you used uppercase L in lists so it was a broken import 👀